src/app/core/user-agent/user-agent.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { environment } from 'environments/environment';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserAgentService {
```

Comment:
That's it? We create a whole new class just to produce a single string taken from an environment? That really seems overkill..
I guess it would be more useful if this `UserAgentService` would create an actual user agent object, not just a string, including setting/overriding sereral headers or request params like TLS...